### DISCUSSION TITLE
PAI sucks!

### DISCUSSION THREAD

**Discussion by YokoTheDuke** [zepp-health/zeppos-forum#320] [User Feedback]
I apologize for the lurid headline. I'm interested in attention because of the PAI algorithm that awards points for sporting activities.

In my opinion something is wrong. In any case, I no longer receive points for some sporting activities.

Yes, I realize that the higher your level of physical fitness you get fewer points, but no points at all is demotivating. Moderate physical activity is also something very positive for the organism.

So, I don't get any points for 2 hours (10K) walking. I don't get any points for a 25 minute HIIT. I don't get any points for a 5K run in 30 minutes. My cardio fitness is 42. I am 60 years old.

If I'm in the wrong place because of my PAI criticism, perhaps one of you knows the right person to contact.

**Comment by MyoungXUE** [comment]
Hello, please follow this step to submit user feedback and tell me your Zepp ID.

https://docs.zepp.com/docs/guides/faq/upload-log/

**Reply by YokoTheDuke** [reply]
Hey MyoungXUE,
 I get your e-Mail and I'll use the form in the ZEPP OS DOC (doc.zepp.com) a.s.a.p. for my correct answer with all necessary details!

Thanks for your awesome fast response!!!

Bests

YTD

**Reply by YokoTheDuke** [reply]
Done! Protocol uploaded with two screenshots from the ZeppApp on my iOS device.

**Comment by YokoTheDuke** [comment]
Hallo MyoungXUE,

I would like to give a short feedback here. I assume that my criticism of no/few PAI points was taken seriously. In any case, I am now getting PAI points again for a 5K run, also in a HIIT. It's not as much as it used to be, but at least I'm getting points again. Recently, at 60, I ran 10K in 1 hour and 3 minutes. That's a good time for me. When I was 40, I sometimes wasn't any faster. I used to get the maximum = 75 PAI, but now I'm happy with the 50 PAI that I get. ;) Thanks for the support.

Best

YTD

**Comment by YokoTheDuke** [comment]
Hello Myoung XUE, me again. 

Yes, in line with my last feedback here to you, I now get a few more PAI than the month before, but unfortunately still not enough. About 20-30% too little, I guess.

Despite frequent exercise, I am unable to reach 100 PAI per week now in the summer. Hard to believe.

It can't be true that at 60 years old and as a private recreational athlete, I apparently have to run 3 marathons to reach PAI points of 120-180 per week like I used to.

Currently, I am almost always UNDER 100 PAI.

So that everyone reading this can understand this, here are the data. See screenshots!

07. JUNI – 10K Run = 28 PAI

08. JUNI –  5K Run = 10 PAI

10. JUNI - 30 min. HIIT = 2 PAI

10. JUNI –  5K Run = 24 PAI 

Highscore 102 PAI

11. JUNI – 30 min. HIIT = 3 PAI

12. JUNI –  5k Walk = 0 PAI (?!?!?)

13. JUNI - 10k Biking = 2 PAI (?!?!)

14. JUNI -  5K Run = 19 PAI

15. JUNI - 10K Run = 33 PAI

16. JUNI – 30 min. HIIT = 1 PAI

17. JUNI –  5K Run = 18 PAI

Highscore 88 PAI

This is really not funny. Why is there no PAI for walking 5K when walking is supposed to be particularly good for the organism?

So, almost every day, I get exercise. I will soon be 61? Are there reasons for this that are not obvious? Is anyone worried that the sporty old people might overdo it?

Is smoking, drinking alcohol and sitting on your butt all day less “dangerous”???
I certainly don't need to have PAI high scores of over 200 PAI in a week like I did 1-2 years ago. But constantly having PAI in the summer, when I can do a lot more sport than in the winter, just above or above 100 PAI - that's really not motivating.

So - if necessary, pass the information on to the medical guys in Norway.

PAI is fine, but it should also motivate and I'm slowly losing my enjoyment of it. I'd rather do without PAI tracking in the future and do it without.

Best

![PAI 1](https://github.com/zepp-health/zeppos-forum/assets/32335666/9a729a28-757c-4163-b7ab-373b313c6b27)
![PAI 2](https://github.com/zepp-health/zeppos-forum/assets/32335666/083ff45a-e18a-4800-a4b9-5e8eefd20179)
